### PR TITLE
Deprecate REST dashboard analytics

### PR DESCRIPTION
**Pull request by jacobbednarz**
Removes the dashboard analytics API calls in favour of a link to the Cloudflare dashboard.

Screencast of the change (there is also an "analytics" header on the page not shown in the screenshot)

Dr2Ja3uS-jUn7RFzG-gZalUJwF

### CODE REVIEW

File: lang/en.js [hunk 1]
```diff
@@ -83,6 +83,9 @@
     "container.analyticCard.ssl.firstText": "SSL secured request",
     "container.analyticCard.ssl.secondText": "unsecured requests",
     "container.analyticCard.ssl.drawer.help": "Total number of encrypted 'HTTPS' requests sent using SSL/TLS.\n\n**Note:** In order to increase the amount of traffic served over HTTPS, create a Page Rule as follows: add a new rule with a URL Pattern (e.g., *yourdomain.com/*) and set Always Use HTTPS to On.",
```
Comment (manatarms) [inline lang/en.js:85]: Lets clean up the translation keys we removed.

File: src/containers/AnalyticsPage/AnaltyicsPage.js [hunk 2]
```diff
@@ -1,453 +1,39 @@
...
+      <div>
+        <Card>
+          <CardSection>
+            <CardContent title={<FormattedMessage id="container.analyticCard.graphql.header" />}>
+              <p>
+                <FormattedMessage id="container.analyticCard.graphql.overview" />
+              </p>
+            </CardContent>
+            <CardControl>
+              <Button type="primary" onClick={function(){window.open(analyticsDeepLink)}}>
```
Comment (manatarms) [inline src/containers/AnalyticsPage/AnaltyicsPage.js:23]: Lets add 'noopener,noreferrer' to protect the parents opener object.

Comment (jacobbednarz) [reply src/containers/AnalyticsPage/AnaltyicsPage.js]: sorted in 13e6fc4

File: lang/en.js (hunk 1 above)
Comment (jacobbednarz) [reply lang/en.js]: updated in b4e37aa

**Review by manatarms — APPROVED** [review]

**Review by garrettgalow — APPROVED** [review]

**Pull request merged**